File: SqlParser/SqlParser/LiteralExpression.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SqlParser
{
    public class LiteralExpression : Expression
    {

        public override string Eval(IExpressionContext ctx)
        {
            StringBuilder sb = new StringBuilder();
            foreach (Expression ex in lists)
            {
                sb.Append(ex.Eval(ctx));
                sb.Append(" ");
            }
            return sb.ToString();
        }
    }
}

File: app-fw/app-base/src/main/java/org/yy/base/dao/dynamic2/LiteralExpression.java
package org.yy.base.dao.dynamic2;

public class LiteralExpression extends Expression {

	public String Eval(ExpressionContext ctx) {
		StringBuilder sb = new StringBuilder();
		for (Expression ex : lists) {
			sb.append(ex.Eval(ctx));
			sb.append(" ");
		}
		return sb.toString();
	}
}
